mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestDerivateContents.java

```diff
@@ -257,19 +258,8 @@ private static String getDigestHeader(String md5sum) {
     }
 
     private static byte[] getMD5Digest(String md5sum) {
-        final char[] data = md5sum.toCharArray();
-        final int len = data.length;
-
-        // two characters form the hex value.
-        final byte[] md5Bytes = new byte[len >> 1];
-        for (int i = 0, j = 0; j < len; i++) {
-            int f = Character.digit(data[j], 16) << 4;
-            j++;
-            f = f | Character.digit(data[j], 16);
-            j++;
-            md5Bytes[i] = (byte) (f & 0xFF);
-        }
-        return md5Bytes;
+        MCRMD5Digest md5 = new MCRMD5Digest(md5sum);
```

Comment:
I'd suggest changing `getDigestHeader(String)` to 

```
private static String getDigestHeader(String md5sum) {
    return "MD5=" + Base64.getEncoder().encodeToString(HexFormat.of().parseHex(md5sum));
}
```

and removing `getMD5Digest` altogether.  Format conversion can be done with standardized Java methods now.

Reply:
applied in https://github.com/MyCoRe-Org/mycore/pull/2328/commits/2286a65a8d52afbbf9d28f008524f244e39dcc05